docs/css/main.css

```diff
@@ -0,0 +1,221 @@
+mark {
+    background-color: #ff0;
+    border-radius: 5px;
+    padding-top: 0;
+    padding-bottom: 0;
+}
+
+.indented {
+    padding-left: 20px;
+}
+
+.theme-card img {
+    width: 100%;
+}
```

Comment:
I couldn't find any usages of this class, is this needed?

Reply:
This part of the css is generated by the `makbind init` command. Best to leave it be.